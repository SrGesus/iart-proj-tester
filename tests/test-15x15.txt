FE	LV	VB	VC	LV	FE	FC	VB	FD	FC	LH	BB	LV	LV	FD
FE	VD	VD	BD	LH	FC	BC	VE	VB	VE	VC	BE	FD	VC	FB
BB	BB	LH	VD	VE	FB	LV	FE	LV	LH	BE	LV	LV	BC	FD
LV	FB	FE	VD	VE	FC	LV	LH	LH	LV	BD	LV	FD	VC	FD
BC	BC	BB	BB	LH	BC	VD	VC	BB	BB	VC	FD	VD	BC	FD
LV	VC	FC	FB	VB	LV	VB	FD	BE	BC	FD	VB	BD	VC	VE
BC	VD	FC	BE	LH	FD	FC	FE	LH	BC	LH	FB	BE	LH	VD
VD	LV	VE	VE	BC	LV	VE	VC	LV	BE	LV	LH	BD	FE	LV
FD	LV	BB	LH	BE	BB	VE	FB	LH	BE	BD	VE	VB	LH	VD
VC	FB	FD	VE	BD	VB	FE	LV	LV	FE	LH	FD	LH	VC	FD
VD	BE	BD	BC	BE	LH	BB	BE	BB	VD	LV	FD	BE	BD	FE
FE	BB	FD	VD	BD	BE	BD	BE	VE	FE	FC	FC	FC	VE	BD
FE	BE	VD	BB	FE	LH	FD	LH	LV	FE	FD	BE	BD	VD	LV
VE	BD	FC	BC	VD	VC	FD	LH	FB	VC	BE	VC	VE	LH	LV
FB	FE	VC	BB	LV	LV	FB	VD	LV	LH	BD	FE	FE	VE	VB
