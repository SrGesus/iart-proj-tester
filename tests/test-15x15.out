FD	LH	VE	VB	LH	FE	FB	VB	FE	FD	LH	BB	LH	LH	FE
FB	VB	VC	BD	LH	FE	BD	VC	VB	VE	VB	BC	FE	VB	FE
BD	BC	LH	VC	VB	FE	LV	FB	LV	LV	BD	LH	LH	BC	FE
LV	FB	FB	VB	VC	FB	LV	LV	LV	LV	BD	LH	FE	VB	FE
BD	BC	BC	BC	LH	BC	VC	VD	BE	BD	VC	FB	VB	BE	FB
LV	VB	FE	FB	VB	LH	VE	FD	BE	BD	FE	VD	BE	VD	VC
BD	VC	FD	BE	LV	FB	FC	FB	LV	BD	LH	FE	BD	LH	VE
VD	LH	VE	VD	BE	LV	VB	VC	LV	BD	LH	LH	BC	FE	LV
FD	LH	BC	LH	BE	BD	VC	FB	LV	BD	BB	VE	VB	LH	VC
VB	FE	FB	VB	BC	VC	FB	LV	LV	FC	LV	FC	LV	VB	FE
VD	BB	BC	BC	BB	LH	BC	BC	BC	VE	LV	FD	BC	BE	FB
FD	BE	FB	VB	BC	BB	BB	BB	VE	FC	FC	FB	FB	VD	BE
FD	BE	VD	BE	FB	LV	FC	LV	LV	FB	FB	BD	BC	VE	LV
VB	BE	FB	BD	VC	VD	FE	LV	FC	VD	BE	VD	VE	LV	LV
FC	FC	VD	BC	LH	LH	FE	VD	LH	LH	BC	FE	FC	VD	VC
